app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -233,8 +233,11 @@ private static void logResponse(int responseCode, String responseString) {
         if (!(responseCode >= 200 && responseCode < 300)) {
             Logger.log(LogTypes.TYPE_WARNING_NETWORK, responseCodeMessage);
             Logger.log(LogTypes.TYPE_FORM_SUBMISSION, responseCodeMessage);
-            Logger.log(LogTypes.TYPE_FORM_SUBMISSION,
-                    "Response string to failed form submission attempt was: " + responseString);
+            if (!responseString.startsWith("<html>")) {
```

Comment:
Don't we still want to log just the response code if not the whole failure message ?

Reply:
Yea we do, that's why I left it outside of this new `if` block. Or are you saying you think we shouldn't? We definitely still want to log the response code whenever submission fails, even if the response _message_ isn't useful.

Reply:
Ahh sorry. Missed that we are already loggin the code.